### PR TITLE
Updated README.md

### LINKED ISSUE #26: Better documentation for non technical users
We had some user feedback around the documentation being hard to understand:
- How the site is built into the `docs` folder
- How the generated files get made by pancake

### PR DESCRIPTION
**Pull request by sukhrajghuman**
fixes #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,54 @@
...
 
 
-## Install
+## Get started
 
-Download, clone, or fork this repository and run `npm install` to install all dependencies.
+The starter pack uses NPM (node package manager) to set up a local development environment and install the Design System components. NPM is a package manager for Node based projects. NPM comes pre-installed with Node.
 
-After you have all dependencies installed, run `npm run watch` to spin up a new local server and watch for changes on `*.scss` files. Changes will be injected into your browser so you don’t even need to refresh thanks to [Browser Sync](https://www.browsersync.io/).
+1. Make sure you have `NPM` and `node` installed on your computer. You can follow the [NPM instructions](https://www.npmjs.com/get-npm) if you are unsure.
```
Comment (Xochi) [inline README.md]: npm should be all lower case where applicable. It is also the default package manager for node. It is not an acronym too.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,54 @@
...
+
+When the user runs `npm run build` or `npm run watch` we use [node-sass]("https://www.npmjs.com/package/node-sass?activeTab=versions") to convert the `src/sass/main.scss` file into the `docs/css/main.css` file. [Autoprefixer]("https://www.npmjs.com/package/autoprefixer") is ran after compilation to add vendor prefixes to the `main.css` file.
+
+> Refer to the `package.json` scripts for how this has been set up
+
+This set up allows you to modify the variables in the Design System file from the `main.scss` file. You can add your own colour scheme, or change the typography and spacing. Tweaks to the components or additional ones should be added below the import of `_uikit.scss`. Changes to these files will be injected into your browser so you don’t even need to refresh thanks to [Browser Sync](https://www.browsersync.io/).
+
+The `docs` folder in the contains all the files required to publish a website. The `index.html` file references the `main.css` file and `script.min.js` assets to generate the page. We name the folder `docs` because [github pages](https://pages.github.com/) uses the docs folder to host static websites.
```
Comment (Xochi) [inline README.md]: This looks great!

Comment (sukhrajghuman) [reply README.md]: @Xochi cheers!  I've made those changes. Let me know what you think :)

**Comment by Xochi** [comment]
looks good @sukhrajghuman

**Pull request merged**